test/blackbox/package-for-blackbox-tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script runs *as part of building a test_subject docker container* to take the current
+# version of the library and package it up with the other dependencies ready for running HTTP
+# tests. It runs in a *composer* image, not our own, and is used as the first part of a
+# multi-stage build. See the docker-compose.yaml for how this fits together.
+
```

Comment:
I would extract or duplicate the bones of this comment and the ones below up to the README as I was pretty confused about where the `git export` was actually running when I was reading it until I found these comments.

Reply:
Yeah was planning to add an overall readme and point to this.